extensions/2.0/Khronos/KHR_materials_variants/schema/mesh.primitive.KHR_materials_variants.mappings.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_variants mesh primitive extension",
+    "type": "object",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "variants": {
```

Comment:
Uniqueness of each mapping is important so I can add this in. Just wondering how JSON schema validators will work if it's an object type with keys instead of scalar values.